### PR TITLE
Javascript nav

### PR DESCRIPTION
**Pull request by KUBAHO3**
In this project:

- I have added the mobile navbar section
- I have added the closing of the navbar
- I have added on click on nav item direct me to the section choosen
- I have fixed all linter errors occured

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,6 +31,14 @@
             </div>
         </nav>
     </header>
+    <nav class="mobile-nav ">
```
Comment (ShadyShawkat) [inline index.html]: - [x] Please make sure that the JS linters are setup correctly, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.) requirement.

File: index.html [hunk 2]
```diff
@@ -31,6 +31,14 @@
...
+        <span class="close-button">x</span><br><br>
+        <ul class="menu-list-mobile" >
+            <li><a class="menu-list-item" href="#works">Portfolio</a></li><br>
+            <li><a class="menu-list-item" href="#about-myself">About</a></li><br>
+            <li><a class="menu-list-item" href="#my-form">Contact</a></li><br>
+        </ul>
+    </nav>
```
Comment (ShadyShawkat) [inline index.html]: - Kindly make sure to stick as much as possible to the Figma template, which includes:
    - [x] Changing the background color of the "close" button and adjusting its size.
    - [x] Applying a background filter to match the original design,

| Your design | Original design |
| --- | --- |
| image | image |

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (KUBAHO3) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (KUBAHO3) [reply index.html]: done

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @KUBAHO3 ,

Your are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- The background filter is added :+1: 
- The close button looks like Figma's
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**